### ISSUE TITLE
cursor bound input not working anymore

### ISSUE THREAD

**Issue by ZiDa20** [ZiDa20/Biophysical_Essentials#145]
![image](https://github.com/ZiDa20/Biophysical_Essentials/assets/15120246/0330e8be-c744-4527-ad03-d2b129775b49)
entering the cursor bound values into the table does not work anymore

**Comment by mouzkolit** [comment]
@ZiDa20  is this still an issue?

**Comment by ZiDa20** [comment]
Yes: cursor bounds (green) were manually entered but not updated in the plot: 

![issue_cursor_bounds](https://github.com/user-attachments/assets/7a38ffb2-17cf-4a88-8c71-34420b59fcb9)

**Comment by mouzkolit** [comment]
will add this to the PR

**Comment by ZiDa20** [comment]
fixed
